kaspresso/src/main/kotlin/com/kaspersky/kaspresso/runner/listener/KaspressoLateRunListener.kt

```diff
@@ -0,0 +1,31 @@
+package com.kaspersky.kaspresso.runner.listener
+
+import org.junit.runner.Description
+
+interface KaspressoLateRunListener : KaspressoRunListener {
+
+    fun lateInit(cache: Cache) {
+        testRunStarted(cache.testRunStartedDescription)
```

Comment:
Once this listener is created and added to the notifier, this method is called with all the cached data from missed events.